src/test/java/tutorease/address/testutil/PersonUtil.java

```diff
@@ -44,6 +44,21 @@ public static String getPersonDetails(Person person) {
         return sb.toString();
     }
 
+    /**
+     * Returns the part of command string for the given {@code person}'s details with role.
```

Comment:
The javaDoc comment should be 

`Returns the part of command string for the given {@code person}'s details without role.`